### ISSUE TITLE
API for default wait time when getting MainWindow

### ISSUE THREAD

**Issue by JohanLarsson** [JohanLarsson/Gu.Wpf.UiAutomation#51]
Now it is 
```cs
public Window MainWindow => this.GetMainWindow(TimeSpan.FromSeconds(10));
```

Which is not very nice and the time is probably too short for real applications.
Static property is an alternative but not very nice.
Another alternative is attribute.

**Comment by JohanLarsson** [comment]
Sounds like a bug, have you seen anything dumb in the code?
https://github.com/GuOrg/Gu.Wpf.UiAutomation/blob/2916d6685b3cd993fd1bebd69b64ef7568551119/Gu.Wpf.UiAutomation/Application.cs#L344-L369

**Comment by davishoang96** [comment]
I did try 
``   
    App.WaitForMainWindow(TimeSpan.FromSeconds(1));
``
Although the time is still longer than 1 secs

**Comment by JohanLarsson** [comment]
Did it timeout when you did `App.WaitForMainWindow(TimeSpan.FromSeconds(1));`?

**Comment by davishoang96** [comment]
Yes it did throw timeout exception

**Comment by JohanLarsson** [comment]
Do you have API ideas for this? Static property `App.WaitForMainWindowTime` is not awesome but perhaps not so bad.

**Comment by davishoang96** [comment]
Yeah I fixed the issue by putting: `Application.WaitForMainWindow(AttachProcess, TimeSpan.FromSeconds(1));` before the `App = Application.Attach(AttachProcess.Id);`

It worked perfectly fine. 
Thanks Johan.
